index.html

```diff
@@ -0,0 +1,217 @@
+            <ul class="dot">              
+                <li>Back End Dev</li>               
+                <li>2015</li>    
+            </ul>
+        </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="skills-card">
+                <li class="skill-1">html</li>
+                <li class="skill-1">css</li>
+                <li class="skill-1">Javascript</li>
+            </ul>
+            <button class="btn"><a href="">See Project</a></button>
+        </article>
+
+      </section>
+      <section class="skills">
+        <article class="cards">
+            <figure>
+                <img src="images/yoga.png" alt="">
+            </figure>
+            <h2 class="title-card">Tonic</h2>    
+        <div class="header-details">
+            <h4>CANOPY</h4>           
+            <ul class="dot">              
+                <li>Back End Dev</li>               
+                <li>2015</li>    
+            </ul>
+        </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="skills-card">
+                <li class="skill-1">html</li>
+                <li class="skill-1">css</li>
+                <li class="skill-1">Javascript</li>
+            </ul>
+            <button class="btn"><a href="">See Project</a></button>
+        </article>
+
+      </section>
+      <section class="skills">
+        <article class="cards">
+            <figure>
+                <img src="images/art.png" alt="">
+            </figure>
+            <h2 class="title-card">Multi-Post
+                Stories</h2>    
+        <div class="header-details">
+            <h4>CANOPY</h4>           
+            <ul class="dot">              
+                <li>Back End Dev</li>               
+                <li>2015</li>    
+            </ul>
+        </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="skills-card">
+                <li class="skill-1">html</li>
+                <li class="skill-1">css</li>
+                <li class="skill-1">Javascript</li>
+            </ul>
+            <button class="btn"><a href="">See Project</a></button>
+        </article>
+      </section>
+
+      <!-- </* ----------- ABOUT MYSELF ----------- */> -->
+
+      <Section class="about-myself">
+        <article class="about">
+        <h2 class="title-card">About Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a class="link" href="#">LET'S CONNECT</a>
+        <ul class="social-icons">
```

Comment:
- [ ] As required , your design should stick to the Figma design , meaning that attributs such as colors, fonts, paddings , margins etc should be exactly as in the Figma template. Noticed , this paragraph looks a bit bigger , Kindly use a font-size of `15px` for it to stick to the Figma design

![Capture3](https://github.com/jerymey/Portfolio-setup/assets/80458199/0b657aa3-46d9-439e-bc6c-f79802a19fee)